conu/backend/docker/container.py

```diff
@@ -167,15 +169,32 @@ def get_ports(self):
             ports.append(p.split("/")[0])
         return ports
 
-    def is_port_open(self, port, timeout=2):
+    def is_port_open(self, port, timeout=10):
```

Comment:
Maybe 10 seconds is too much for yes/no question type of method.

Reply:
agreed